route/build/src/public_var.f90

```diff
@@ -101,9 +101,14 @@ MODULE public_var
   logical(lgt)         ,public    :: is_flux_wm           = .false.         ! logical if flow is added or removed from a reach
   logical(lgt)         ,public    :: is_vol_wm            = .false.         ! logical if target volume is considered for a lake
   logical(lgt)         ,public    :: is_vol_wm_jumpstart  = .false.         ! logical if true the volume is reset to target volume for the first time step of modeling
-  logical(lgt)         ,public    :: suppress_runoff      = .false.         ! logical to suppress the read runoff to zero(0)
-  logical(lgt)         ,public    :: suppress_P_Ep        = .false.         ! logical to suppress evaporation and precipitation to zero(0)
-  logical(lgt)         ,public    :: compWB               = .false.          ! logical if entire domain water balance is computed
+  real(dp)             ,public    :: scale_factor_runoff  = realMissing     ! float scale to scale the runoff
+  real(dp)             ,public    :: offset_value_runoff  = realMissing     ! float offset for runoff
+  real(dp)             ,public    :: scale_factor_Ep      = realMissing     ! float scale to scale the potential evaporation
+  real(dp)             ,public    :: offset_value_Ep      = realMissing     ! float offset for evaporation
+  logical(lgt)         ,public    :: flip_Ep              = .false.         ! set to true when convention of upward evaporation is negative
```

Comment:
is_flip_Ep is nicely defined here, so it's OK. But, I wonder if there is a better name for it? Ep_upward? Or Ep_upward_convention? Names are hard to get right, but so good when they are done well.

Reply:
Should it also start with "is_" prefix so that it's consistent with other logicals?

Reply:
The flag is renamed to `is_Ep_upward_negative`. do you think it is a good choice? @ekluzek

Reply:
I can understand what it means, just by the variable name. So I think that's great.